Babylon/commands/macro/destroy.py

```diff
@@ -83,6 +85,15 @@ def destroy(state: dict, azure_token: str, state_to_destroy: pathlib.Path):
     env.store_state_in_local(state=state)
     env.store_state_in_cloud(state=state)
 
+    # deleting web app
+    webapp_id = state['services']['webapp']['webapp_name']
+    logger.info(f"Deleting webapp {webapp_id} ....")
+    azure_token = get_azure_token()
+    subscription_id = state['services']['azure']["subscription_id"]
+    swa_svc = AzureSWAService(azure_token=azure_token, state=state['services'])
+    swa_svc.delete(webapp_name=webapp_id, force_validation=True)
+    state['services']['app']["object_id"] = ""
```

Comment:
I think we are wrong about this, we are not deleting the good thing from state, instead of this line we must have:
state['services']['webapp']["name"] = ""
state['services']['webapp']["static_domain"] = ""

and don't forget to add:
        env.store_state_in_local(state)
        env.store_state_in_cloud(state)
:slightly_smiling_face: